app/jobs/integration_customers/create_job.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module IntegrationCustomers
+  class CreateJob < ApplicationJob
+    queue_as 'integrations'
+
+    retry_on LagoHttpClient::HttpError, wait: :exponentially_longer, attempts: 3
+
+    def perform(integration_customer_params:, integration:, customer:)
+      result = IntegrationCustomers::CreateService.call(params: integration_customer_params, integration:, customer:)
```

Comment:
I'm surprised to see `CreateJob` and `UpdateJob` call `CreateService` and `UpdateService` but no reference to the new `CreateOrUpdateService` 🤔

Reply:
That was the goal:) Because of it, new logic can be merged and we will just open new PR later with the tiny change where `CreateOrUpdate` is actually called. That approach allowed us not to carry many branches. Instead all new logic got extracted and merged and later we will just call new logic where needed.